File: Wander.Server/Model/Forms/UserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Wander.Server.Model
{
    public class UserModel
    {       
        public string Login { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public int Sex { get; set; }
    }
}

File: Wander.Android.App/core/src/com/wander/game/models/UserModel.java
package com.wander.game.models;

/**
 * Created by valentin on 07/12/2015.
 */
public class UserModel
{
    @com.google.gson.annotations.SerializedName("Login")
    public String Login;
    @com.google.gson.annotations.SerializedName("Email")
    public String Email;
    @com.google.gson.annotations.SerializedName("Password")
    public String Password;
    @com.google.gson.annotations.SerializedName("Sex")
    public int Sex;
}
